### DISCUSSION TITLE
The profile of the project creator was hacked

### DISCUSSION THREAD

**Discussion by brunohash** [bet-blocker/bet-blocker#21] [General]
Today, around 09:00 BRT, I noticed that the profile of Jonathan Henrique, CTO of ApiBrasil, was hacked. Instead of his information, there was data from a Korean architect. Along with some friends, we commented on some posts that the hacker had put up, and he deleted the publications. The strangest thing is that they only deleted the post about the DNS blocking project.

![image](https://github.com/user-attachments/assets/05db2aaf-5875-4a23-891a-9d5f226ad20a)

Guys, don’t be alarmed, we will investigate to understand what happened. In any case, this means we are on the right track.

**Comment by RobsonTrasel** [comment]
It's sad how this happened, this project is based on getting more people to stop throwing their lives and money away. You have my full support for any further problems.

**Reply by jhowbhz** [reply]
Hi Robson

Thank you for your support and for your repeated attempts to warn me about the attack. Thank you. Let's work together to help those who want to get out of this hole.

**Comment by jhowbhz** [comment]
> Hoje, por volta das 09:00 BRT, notei que o perfil do Jonathan Henrique, CTO da ApiBrasil, foi hackeado. No lugar das informações dele, havia dados de um arquiteto coreano. Junto com alguns amigos, comentamos alguns posts que o hacker havia feito, e ele apagou as publicações. O mais estranho é que eles só apagaram o post sobre o projeto de bloqueio de DNS.
> 
> ![imagem](https://private-user-images.githubusercontent.com/15908424/384432149-05db2aaf-5875-4a23-891a-9d5f226ad20a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzEwOTU3MDYsIm5iZiI6MTczMTA5NTQwNiwicGF0aCI6Ii8xNTkwODQyNC8zODQ0MzIxNDktMDVkYjJhYWYtNTg3NS00YTIzLTg5MWEtOWQ1ZjIyNmFkMjBhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTA4VDE5NTAwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZkOTJjMGUxMzJiY2I1NTM3NjA3NmYyMzU5YjY0NWVkZGVkYmI1Y2Q1N2Y1MGU1N2E5NjE1OWMwYzllNDk1NTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.t8bydEh_RAiFKANzIeiIViI5x1ndiuLQJmprFFrxA2g)
> 
> Pessoal, não se assustem, vamos investigar para entender o que aconteceu. De qualquer forma, isso significa que estamos no caminho certo.

Hi Bruno,

Thank you very much for your support, the project remains intact, this attack only made me want to continue following it and learn more about the subject, today the unbridled gambling has ended many lives and it is time to try to help those who want to get out of this hole!

Thank you for your support always!
